autometrics/tests/custom_histogram_buckets_test.rs

```diff
@@ -12,6 +12,7 @@ fn custom_histogram_buckets() {
     AutometricsSettingsBuilder::new()
         .histogram_buckets(vec![0.1, 0.2, 0.3, 0.4, 0.5])
         .init();
+    prometheus_exporter::init();
```

Comment:
Should initializing the settings also initialize this (if the `prometheus-exporter` feature flag is enabled)? It's important that the settings are configured _before_ the `prometheus_exporter` is initialized, so it seems like we should try to remove this as a footgun